t/e2e/api/api_test.go

```diff
@@ -274,3 +274,20 @@ func (s *APITestSuite) TestNodeStartCrash() {
 	// cleanup
 	s.NoError(s.api.StopNode())
 }
+
+func (s *APITestSuite) TestStatusGoVersion() {
+	var (
```

Comment:
`var` is not needed, `require := s.Require()` will work just fine.

Also, I don't think you need to start the node here. 

```go
version := s.api.StatusGoVersion()
s.Equal(params.Version, version)
```

should be enough.